test/Datadog.Trace.ClrProfiler.IntegrationTests/SmokeTests/StackExchangeRedisAssemblyConflictLegacyProjectSmokeTest.cs

```diff
@@ -0,0 +1,29 @@
+using Datadog.Core.Tools;
+using Datadog.Trace.ClrProfiler.IntegrationTests.Helpers;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.SmokeTests
+{
+    public class StackExchangeRedisAssemblyConflictLegacyProjectSmokeTest : SmokeTestBase
+    {
+        public StackExchangeRedisAssemblyConflictLegacyProjectSmokeTest(ITestOutputHelper output)
+            : base(output, "StackExchange.Redis.AssemblyConflict.LegacyProject", maxTestRunSeconds: 15)
+        {
+        }
+
+        [TargetFrameworkVersionsFact("net452;net461")]
```

Comment:
Nice, didn't know that was a thing.

Reply:
@zacharycmontoya wrote it! It's a specialized `[Fact]` that sets the `Skip` property if the target framework doesn't match.

Reply:
💯 @zacharycmontoya 👍

Reply:
Oops, I meant to include a link to [`TargetFrameworkVersionsFact`](https://github.com/DataDog/dd-trace-dotnet/blob/58908e43b3e1d5da3bcd19499709da6b2b259a83/test/Datadog.Trace.ClrProfiler.IntegrationTests/Helpers/TargetFrameworkVersionsFact.cs).